File: src/main/java/io/github/svegon/capi/mixininterface/IStatHandler.java
package io.github.svegon.capi.mixininterface;

import it.unimi.dsi.fastutil.objects.Object2IntMap;
import net.minecraft.stat.Stat;
import net.minecraft.stat.StatHandler;

public interface IStatHandler {
    Object2IntMap<Stat<?>> getStatMap();

    void copyFrom(StatHandler statHandler);
}

File: src/main/kotlin/io/github/svegon/mclientapi/mixininterface/IStatHandler.kt
package io.github.svegon.mclientapi.mixininterface

import it.unimi.dsi.fastutil.objects.Object2IntMap
import net.minecraft.stat.Stat
import net.minecraft.stat.StatHandler

interface IStatHandler {
    val statMap: Object2IntMap<Stat<*>>

    fun copyFrom(statHandler: StatHandler)
}
